tools/dawgrun/pkg/go-repl/Makefile

```
@@ -0,0 +1,15 @@
+all: ./examples/basic_repl
```

checkmake (0.3.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)